core/src/main/java/hudson/model/Job.java

```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- * 
+ *
```

Comment:
Please do not include unrelated whitespace changes.

Reply:
Oops, Intellij did it. (I remember I rolled back those lines, but they comes up again after I push...)